core/src/main/java/org/mule/runtime/core/api/util/ClassUtils.java

```diff
@@ -860,7 +861,10 @@ public static void withContextClassLoader(ClassLoader classLoader, Runnable runn
    * @param f the function to memoize
    * @param cache the map where cached values are stored
    * @return the memoized function
+   *
+   * @deprecated use {@link FunctionalUtils#memoize(Function, Map)}
```

Comment:
ClassUtils is API, and FunctionUtils isn't, so this isn't really a valid deprecation message is it?